server/src/app.py

```diff
@@ -42,10 +41,10 @@ async def show_calendar():
     return calendar_handler.get_calendar_json()
 
 
-@app.post("/files", status_code=200)
-async def get_deadlines(response: Response, outline_files: List[UploadFile] = File(...)):
+@app.post("/file", status_code=200)
```

Comment:
This should still be `/files`. 

It was plural not because the endpoint accepts multiple files. It's by the best practices of RESTful API, to name _collections_ of objects with plural nouns.